ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/SketchOnMap/README.metadata.json

```diff
@@ -1,7 +1,7 @@
 {
     "category": "Display information",
     "description": "This sample demonstrates how to use the Sketch Editor to edit or sketch a new point, line, or polygon geometry on to a map.",
-    "featured": true,
+    "featured": false,
```

Comment:
I'm marking the SketchEditor samples as no longer featured because we'd rather showcase the GeometryEditor